### PR TITLE
Allow to comment out lines toolbar setting

### PR DESCRIPTION
**Pull request by ocorreiododiogo**
This change would allow people to "comment out" lines in the toolbar setting by inserting a # at the beginning.

### CODE REVIEW

File: wire/modules/Inputfield/InputfieldCKEditor/InputfieldCKEditor.module [hunk 1]
```diff
@@ -160,7 +161,9 @@ class InputfieldCKEditor extends InputfieldTextarea {
 		$lines = explode("\n", $str); 
 		foreach($lines as $line) {
 			$line = trim($line);
-			if(empty($line)) {
+			if (strpos($line, '#') === 0) {
```
Comment (ocorreiododiogo) [inline wire/modules/Inputfield/InputfieldCKEditor/InputfieldCKEditor.module]: check if the line starts with #, and if so, continue to the next

File: wire/modules/Inputfield/InputfieldCKEditor/InputfieldCKEditor.module [hunk 2]
```diff
@@ -601,7 +604,7 @@ class InputfieldCKEditor extends InputfieldTextarea {
 		$f->attr('name', 'toolbar'); 
 		$f->attr('value', $this->toolbar);
 		$f->label = $this->_('CKEditor Toolbar'); 
-		$f->description = $this->_('Separate each toolbar item with a comma. Group items by placing them on the same line and use a hyphen "-" where you want a separator to appear within a group. If you want more than one toolbar row, separate each row with a blank line.'); // Toolbar options description
+		$f->description = $this->_('Separate each toolbar item with a comma. Group items by placing them on the same line and use a hyphen "-" where you want a separator to appear within a group. If you want more than one toolbar row, separate each row with a blank line. Insert a hash '#' at the beginning of a line if you want it to be ignored.'); // Toolbar options description
```
Comment (ocorreiododiogo) [inline wire/modules/Inputfield/InputfieldCKEditor/InputfieldCKEditor.module]: Update the instructions in the editor to reflect this change

**Comment by ryancramerdesign** [comment]
I have added this and it will appear in today's commits. Thanks.

**Comment by ocorreiododiogo** [comment]
Nice :)

**Comment by isellsoap** [comment]
@ocorreiododiogo Can this be closed? Please look also to other PRs or issues of you (if there are any), that could be closed. Thanks!

**Comment by isellsoap** [comment]
@ocorreiododiogo ?

**Comment by LostKobrakai** [comment]
@ocorreiododiogo And just another gently reminder to close the PR :)

**Comment by ocorreiododiogo** [comment]
sorry guys, didn't see this

**Pull request closed without merging**